include/aspect/simulator.h

```diff
@@ -333,6 +333,15 @@ namespace aspect
 
     private:
 
+      /**
+       * A member variable that tracks whether we are completely done
+       * with initialization and have started the time loop. This
+       * variable needs to be the first one that is initialized in
+       * the constructor, so that its value correctly tracks the
+       * status of the overall object. As a consequence, it has to
+       * be the first one declared in this class.
+       */
+      bool simulator_is_initialized;
```

Comment:
this is confusing because I expect the function simulator_is_initialized to return this bool.

Reply:
Yes. The "simulator" in `SimulatorAccess::simulator_is_initialized()` doesn't talk about the `Simulator` object itself, but whether the `simulator` member variable (a pointer) of the `SimulatorAccess` object has been set to something non-`nullptr`.

Reply:
Rene agrees. He thinks it would be best to align the name of the variable with the function that actually returns the value.